src/main/java/com/nike/cerberus/operation/composite/CompositeOperation.java

```diff
@@ -119,16 +96,24 @@ public boolean isRunnable(T command) {
 
             // If the given command is not runnable return false for the whole chain
             //noinspection unchecked
-            if (! operation.isRunnable(chainedCommand)) {
-                return false;
+            boolean isRunnable = operation.isRunnable(chainedCommand);
+            if (! isRunnable) {
+                if (! skipOnNotRunnable()) {
+                    throw new RuntimeException("The command: " + chainedCommand.getCommandName() + " is not runnable, breaking the chain");
```

Comment:
Exception message shouldn't be "breaking the chain", It should be more like "The subcommand ... is not runnable, stopping..."

Reply:
- [ ] Done.